dotNetInstallerLib/ProcessComponent.cpp

```diff
@@ -40,10 +40,7 @@ bool ProcessComponent::IsExecuting() const
 
 void ProcessComponent::Wait(DWORD /* tt */)

```

Comment:
I've updated the pull request to address this issue.

The updated pull request also has some other minor fixes like initializing uninitialized variables.

Thank you.